pygef/been_jefferies/util.py

```diff
@@ -24,39 +25,26 @@ def excess_pore_pressure_ratio(df):
     return df
 
 
-def ic_to_gamma(df, water_level):
+def ic_to_gamma(water_level):
     """
-    Assign the right gamma (unit soil weight kN/m^3) to the corresponding Ic.
```

Comment:
Isn't it a good idea to keep this comment?

Reply:
Right, was to hasty.